tests/migration/migration.go

```diff
@@ -2725,7 +2725,19 @@ var _ = SIGMigrationDescribe("VM Live Migration", decorators.RequiresTwoSchedula
 
 			By("Creating ResourceQuota with enough memory for the vmi but not enough for migration")
 			resourceQuota := newResourceQuota(resourcesToLimit, testsuite.GetTestNamespace(vmi))
-			_ = createResourceQuota(resourceQuota)
+			resourceQuota = createResourceQuota(resourceQuota)
+			Eventually(func(g gomegatypes.Gomega) error {
+				quota, err := virtClient.CoreV1().ResourceQuotas(resourceQuota.Namespace).Get(context.TODO(), resourceQuota.Name, metav1.GetOptions{})
+				if err != nil {
+					return err
```

Comment:
Why not Expect?

Reply:
nit: I'd use an except here to avoid retries